ui/decredmaterial/editor.go

```diff
@@ -26,6 +30,24 @@ type Editor struct {
 	shaper text.Shaper
 }
 
+type EditorCustom struct {
+	theme *Theme
+	//title is the title of the editor input field
+	title     string
```

Comment:
Why even have a comment if the field is not exported? Is there a reason it is not exported?

Reply:
Yes. Because the title does uses the hint text as its value. It does not need its value changed or written to externally.

Reply:
Over there the hint (placeholder) can be different from the title.